src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Figure implements Draw {
```

Comment:
```suggestion
public abstract class Figure implements Draw, AreaCalculator {
```
якщо ми імплементуємо інтерфейси в абстрактному класі, то в усіх класах наслідниках можна прибрати імплементацію, методи будуть і так доступні